elasticache/variables.tf

```diff
@@ -3,7 +3,6 @@ variable "environment" {}
 variable "vpc_id" {}
 variable "vpc_cidr" {}
 variable "subnet_ids" { type = list(string) }
-variable "availability_zones" { type = list(string) }
```

Comment:
we're using `availability_zones` in https://github.com/dbl-works/terraform/pull/77.

Let's not remove it here to avoid merge conflicts :)